.claude/commands/multi-agent/qk-tickets.md

```
@@ -0,0 +1,142 @@
+  - Bash
+  - Read
+  - Agent
+---
+
+# List Available Tickets
+
+Query requirements.db for approved tickets ready to work on.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)